cohort_4/playbook/activities.md

```diff
@@ -24,6 +29,51 @@
 
 TODO
 
+## Brainstorming
+
+### Forming groups
+
+- Assign people to groups.
+  - There are social pressures between friends when forming groups. Start by
+    assigning the hackers to random groups.
+  - If any problems arise in a group handle it on a case-by-case basis.
+
+### Yes-and sessions
+
+- In a "Yes-and" session, pairs of hackers build off each-others' ideas to create
+  long, completely nonsensical ideas
+  - "Yes-and" sessions are exercises that help people get out of a judgmental
+    mindset before brainstorming
+- To run a "Yes-and" session, start with an example between facilitators
```

Comment:
Perhaps link to a resource on running "yes and" sessions? I know we've had trouble with using videos to communicate information in the past.